internal/daysteps/daysteps.go

```diff
@@ -12,9 +25,52 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	input := strings.Split(data, ",")
+	if len(input) != 2 {
+		return 0, 0, errIncorrectInput
+	}
+
+	steps, err := strconv.Atoi(input[0])
+	if err != nil {
+		return 0, 0, err
```

Comment:
Ошибки лучше всегда оборачивать
```suggestion
		return 0, 0, fmt.Errorf("invalid steps format: %w", err)
```